distribution/std-lib/Standard/src/Table/Data/Column.enso

```diff
@@ -380,10 +388,66 @@ type Column
 
     ## UNSTABLE
 
-       Returns a column truncated to at most `max_rows` rows.
+       Returns a column containing the first `count` elements in this column.
+
+       Arguments:
+       - count: The number of elements to take from the start of this column.
+
+       If `this` has a number of items in it less than `count`, the entire
+       column will be returned.
     take_start : Integer -> Column
-    take_start max_rows =
-        Column (this.java_column.slice 0 max_rows)
+    take_start count =
+        Column (this.java_column.slice 0 count)
+
+    ## UNSTABLE
+
+       Returns a column containing the last `count` elements in this column.
+
+       Arguments:
+       - count: The number of elements to take from the end of this column.
+
+       If `this` has a number of items in it less than `count`, the entire
+       column will be returned.
+    take_end : Integer -> Column
+    take_end count =
+        start_point = Math.max (this.length - count) 0
+        Column (this.java_column.slice start_point count)
+
+    ## UNSTABLE
+
+       Returns the first element in the column, if it exists.
+
+       If the column is empty, this method will return a dataflow error
+       containing `Nothing`.
+    first : Any ! Nothing
```

Comment:
Why do we throw `Nothing` instead of `Index_Out_Of_Bounds_Error` or something similar?

I'm not sure if we have consensus here, but I thought we were moving away from throwing Nothing, as it is very unhelpful for users especially in the IDE - they get a dataflow error `Nothing` and no information on what has happened.

Reply:
I just wanted it to be uniform with `Vector`. What do you think?

Reply:
I see, I think consistency is a good idea, longer term we should probably move to meaningful errors (at least I think that this was our consensus, if not, we should discuss that probably). But since throwing `Nothing` is present in many places maybe that can be fixed later, whatever you think is better given current time constraints (but let's at least create an issue to not forget that).

Reply:
Fixes are tracked in #1630.